_posts/2019-04-16-preservation.md

```diff
@@ -0,0 +1,358 @@
+---
+date: 2019-04-16 8:56:00 -0400
+title: Preservation
+layout: default
+---
+
+## Preservation
+**by Trey Pendragon**
+
+Lately we've been working on preservation within
+[Figgy](https://github.com/pulibrary/figgy) and have come up with a strategy we
+think works for us. However, in doing so, we've evaluated a few options and come
+up with some assumptions which could be useful to others.
```

Comment:
Add something here about how this isn't our entire preservation story, just comparing some technical storage details.